lib/diff.mli

```diff
@@ -17,8 +17,18 @@ type module_ = {
   mdiff : (Types.module_declaration, module_modification) t;
 }
 
+and modtype = {
+  mtname : string;
+  mtdiff : (Types.modtype_declaration, module_modification) t;
+}
+
 and module_modification = Unsupported | Supported of sig_item list
```

Comment:
I think that this should have been named `signature_modification` and now that module types uses it, it is more visible.

Can you rename it to `signature` ?

(Both a module type and a module have a signature, which is a list of signature items)